OpenEdXMobile/build.gradle

```diff
@@ -233,6 +234,11 @@ dependencies {
 
     // test project configuration
     testImplementation 'junit:junit:4.12'
+    testImplementation 'androidx.test.ext:junit-ktx:1.1.1'
+    testImplementation 'androidx.test:core-ktx:1.2.0'
+    testImplementation 'androidx.arch.core:core-testing:2.1.0'
```

Comment:
are we need these dependencies?

Reply:
Yes we need these dependencies for ViewModel testing